README.md

```diff
@@ -1,5 +1,8 @@
 # Databricks SQL Connector for Python
 
+[![PyPI](https://img.shields.io/pypi/v/databricks-sql-connector?style=flat-square)](https://pypi.org/project/databricks-sql-connector/)
+[![Downloads](https://pepy.tech/badge/databricks-sql-connector)](https://pepy.tech/project/databricks-sql-connector)
```

Comment:
as we can't add the python version to the banner, could you please add the supported python version directly to the readme text:

something like this:

## Requirements
Python x is required

Reply:
Fixed in [dde1e3f](https://github.com/databricks/databricks-sql-python/pull/9/commits/dde1e3f5f0428d5eefb882a4a4762c55f817262c)